mocks/com/cinergix/mapper/data/SimpleUserMock.java

```diff
@@ -0,0 +1,30 @@
+package com.cinergix.mapper.data;
+
+import com.cinergix.mapper.annotation.ResultField;
+import com.cinergix.mapper.annotation.ResultMapped;
+
+@ResultMapped
+public class SimpleUserMock {
+
+	@ResultField( fieldName = "user_name" )
```

Comment:
We should try to get rid of the user of "fieldName" as a requirement. If we can specify the annotation like ```@ResultField( "user_name" )``` it would be ideal for the user